src/components/About.js

```diff
@@ -43,6 +43,8 @@ class AboutDialog extends React.Component {
       apiVersion = `${oVirtApiVersion.get('major')}.${oVirtApiVersion.get('minor')}`
     }
 
+    const message = (<span dangerouslySetInnerHTML={{ __html: `${msg.documentation({ link: `<strong><a href=${fixedStrings.DOCUMENTATION_LINK} target='_blank' id=${idPrefix}-documentation-link>Documentation</a></strong>` })}` }} />)
```

Comment:
Need quotes around a few more attributes on the `a` tag...
```jsx
    const docLink = msg.aboutDialogDocumentationLink({ 
      link: `<strong><a href='${fixedStrings.DOCUMENTATION_LINK}' target='_blank' id='${idPrefix}-documentation-link'>${msg.aboutDialogDocumentationText()}</a></strong>` 
    })
```

and later, put the `span` more inline with the other markup
```jsx
    { fixedStrings.DOCUMENTATION_LINK &&
      <li id={`${idPrefix}-documentation`}>
        <span dangerouslySetInnerHTML={{ __html: docLink }} />
      </li>
    }
```

also, need to pull "Documentation" out of the static text and add a new key for it...